src/axom/inlet/tests/inlet_object.cpp

```diff
@@ -147,6 +147,82 @@ TYPED_TEST(inlet_object, simple_array_of_struct_verify_reqd)
   EXPECT_FALSE(inlet.verify());
 }
 
+TYPED_TEST(inlet_object, simple_array_of_struct_verify_empty)
+{
+  std::string testString = "foo = { }";
+  DataStore ds;
+  Inlet inlet = createBasicInlet<TypeParam>(&ds, testString);
+
+  auto& arr_table = inlet.addGenericArray("foo");
+  // Even though these are required, the input file should still
+  // "verify" as the array is empty
```

Comment:
That seems a bit unexpected.  

(The comment is helpful here, and I suppose I get the logic that `required` is applied to each struct, and this one has none).